base_layer/core/src/validation/block_validators.rs

```diff
@@ -111,7 +111,7 @@ fn check_accounting_balance<B: BlockchainBackend>(
     block
         .body
         .validate_internal_consistency(&offset, total_coinbase, factories)
-        .map_err(ValidationError::from)
+        .map_err(|e| ValidationError::TransactionError(TransactionValidationError::TransactionError(e)))
```

Comment:
This is getting a bit unwieldy.

Let's just collapse the TX errors here:

`.map_err(|e| ValidationError::TransactionValidationError(e.to_string))`